### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by tsenart**
This commit updates the README with project status information and
testing and license sub-sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 Very early version of a pure Go language bindings for Apache Mesos. As with other pure implementation, mesos-go uses the HTTP wire protocol to communicate directly with  a running Mesos master and its slave instances. One of the objectives of this project is to provide an idiomatic Go API that makes it super easy to create Mesos frameworks using Go. 
 
 [![Build Status](https://travis-ci.org/mesos/mesos-go.svg)](https://travis-ci.org/mesos/mesos-go) [![GoDoc] (https://godoc.org/github.com/mesos/mesos-go?status.png)](https://godoc.org/github.com/mesos/mesos-go)
 
-## Current Status
-This is a very early version of the project.  Howerver, here is a list of things that works so far:
+## Status
+This project is undergoing a *complete* rewrite happening in the
+[next](https://github.com/mesos/mesos-go/tree/next) branch. These bindings will
+integrate exclusively with the new public Mesos HTTP API.
```
Comment (ConnorDoyle) [inline README.md]: How about support for custom executors, will the libprocess version be carried over into `next` or will this have to wait until there is a new HTTP API available in Mesos?

Comment (tsenart) [inline README.md]: That's a good question. We have to understand what the timeline for Mesos to have the executor API out is. In any case, the current version of the bindings would still be available.

**Comment by jdef** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (jimenez) [inline README.md]: The executor HTTP API is aimed for Mesos 0.26. So we can integrate it as soon as it makes it to master.

Comment (ConnorDoyle) [inline README.md]: Cool, thanks Isabel.

Comment (tsenart) [inline README.md]: That answers it. No libprocess in the new bindings. Pure HTTP APIs based.

**Pull request merged**